src/IFrameNavigator.ts

```diff
@@ -177,13 +234,13 @@ export default class IFrameNavigator implements Navigator {
 
         let statusMessage = "";
         if (status === CacheStatus.Uncached) {
-            statusMessage = "Not available offline";
+            statusMessage = "Download";
```

Comment:
I guess the status message could replace the button, and not be there when the button is showing. I don't think we need labels for the view and font size controls.